src/Projekt/Main.fs

```diff
@@ -50,7 +50,16 @@ let main argv =
     | MoveFile data ->
         (Project.moveFile data.ProjPath data.FilePath data.Direction data.Repeat)
         |> saveOrPrintError data.ProjPath
-        
+
+    | ListFiles data ->
+        match (Project.listFiles data.ProjPath) with
+        | Success files ->
+            List.iter (fun file -> eprintfn "%s" file) files
```

Comment:
You're right it should be printfn. I think I just used eprintfn since nearby calls were using it and I didn't stop and think about it.  I have updated the fork to use printfn.